### ISSUE TITLE
What about angular5 support?

### ISSUE THREAD

**Issue by munrocket** [anychart-integrations/anychart-angular4#2]
Do you planning to create anychart-angular5 library or example of usage?

**Comment by Shestac92** [comment]
@munrocket 
yes, we are planning to publish an example of usage with angular5. I must say that using angular 5 with AnyChart is absolutely similar to angular 4. You may download our demo for angular 4 and update angular dependencies in package.json up to the 5th version. Or you can wait for publishing new example.
